packages/alist/build.sh

```diff
@@ -2,42 +2,54 @@ TERMUX_PKG_HOMEPAGE=https://alist-doc.nn.ci
 TERMUX_PKG_DESCRIPTION="A file list program that supports multiple storage"
 TERMUX_PKG_LICENSE="AGPL-V3"
 TERMUX_PKG_MAINTAINER="2096779623 <admin@utermux.dev>"
-TERMUX_PKG_VERSION="2.6.4"
+TERMUX_PKG_VERSION="3.0.1"
 TERMUX_PKG_SRCURL=https://github.com/alist-org/alist/archive/v${TERMUX_PKG_VERSION}.tar.gz
-TERMUX_PKG_SHA256=3860f5cec5d809159c55498874c472386cd6142a932a5e7bbfb3b0f569e3e12b
-TERMUX_PKG_REVISION=1
+TERMUX_PKG_SHA256=1f8b0ae87044e161f1fb880bc70bb457424bb9fc0299ac09dc52d773ea6e03e5
 TERMUX_PKG_BUILD_IN_SRC=true
 TERMUX_PKG_AUTO_UPDATE=true
 
-termux_step_pre_configure() {
-	# Switch to local to prevent some countries from not connecting to cdn.
-	sed -i 's|https://npm.*/dist|/|g' ./conf/config.go
-}
-
 termux_step_make() {
 	termux_setup_golang
 
 	# Get alist-web:
 	local ldflags webTag
 	webTag=$(
-		wget -qO- -t1 -T2 "https://api.github.com/repos/alist-org/web-v2/releases/latest" \
+		wget -qO- -t1 -T2 "https://api.github.com/repos/alist-org/alist-web/releases/latest" \
 			| grep "tag_name" | head -n 1 | awk -F ":" '{print $2}' | sed 's/\"//g;s/,//g;s/ //g'
 	)
-	wget https://github.com/alist-org/web-v2/releases/download/${webTag}/dist.tar.gz
+	wget https://github.com/alist-org/alist-web/releases/download/${webTag}/dist.tar.gz
 	tar -zxvf dist.tar.gz
-	rm -f dist.tar.gz
-	sed -ri 's|(lang=")zh-CN"|\1en-US"|g' dist/index.html
-	mv dist/* public/
+	rm -rf public/dist
+	mv -f dist public
+	rm -rf dist.tar.gz
+
+	local _builtAt=$(date +'%F %T %z')
+	local _goVersion=$(go version | sed 's/go version //')
+	local _gitAuthor="Noah Hsu <i@nn.ci>"
+	local _gitCommit=$(git ls-remote https://github.com/alist-org/alist refs/tags/v3.0.1 | head -c 7)
+	export CGO_ENABLED=1
 
-	ldflags="-w -s \
-		-X 'github.com/Xhofe/alist/conf.BuiltAt=$(date +'%F %T %z')' \
-		-X 'github.com/Xhofe/alist/conf.GoVersion=$(go version | sed 's/go version //')' \
-		-X 'github.com/Xhofe/alist/conf.WebTag=${webTag}' \
-		-X 'github.com/Xhofe/alist/conf.GitTag=v${TERMUX_PKG_VERSION}'
-		"
-	go build -o "${TERMUX_PKG_NAME}" -ldflags="$ldflags" -tags=jsoniter alist.go
+	ldflags="\
+	-w -s \
+	-X 'github.com/alist-org/alist/v3/internal/conf.BuiltAt=$_builtAt' \
+	-X 'github.com/alist-org/alist/v3/internal/conf.GoVersion=$_goVersion' \
+	-X 'github.com/alist-org/alist/v3/internal/conf.GitAuthor=$_gitAuthor' \
+	-X 'github.com/alist-org/alist/v3/internal/conf.GitCommit=$_gitCommit' \
+	-X 'github.com/alist-org/alist/v3/internal/conf.Version=$TERMUX_PKG_VERSION' \
+	-X 'github.com/alist-org/alist/v3/internal/conf.WebVersion=$webTag' \
+	"
+	go build -o "${TERMUX_PKG_NAME}" -ldflags="$ldflags" -tags=jsoniter
 }
 
 termux_step_make_install() {
 	install -Dm700 ./"${TERMUX_PKG_NAME}" "${TERMUX_PREFIX}"/bin
```

Comment:
Don't forget to dummy files like `$TERMUX_PREFIX/share/bash-completion/completions/alist.bash`.